README.md

```diff
@@ -71,14 +71,18 @@ Get all the device data messages.
 
 ```javascript
 const KNoTCloudStorage = require('@cesarbr/knot-cloud-sdk-js-storage');
-const client = new KNoTCloudStorage({
-  protocol: 'https',
-  hostname: 'data.knot.cloud',
-  id: 'b1a1bd58-c3ef-4cb5-82cd-3a2e0b38dd21',
-  token: '3185a6c9d64915f6b468ee8043df4af5f08e1933',
-});
 
-client.listData();
+async function main() {
+    const client = new KNoTCloudStorage({
+      protocol: 'https',
+      hostname: 'data.knot.cloud',
+      id: 'b1a1bd58-c3ef-4cb5-82cd-3a2e0b38dd21',
+      token: '3185a6c9d64915f6b468ee8043df4af5f08e1933',
+    });
+
+    console.log(await client.listData());
```

Comment:
Shouldn't we surround it by a `try-catch`? See the example in authenticator SDK: https://github.com/CESARBR/knot-cloud-sdk-js-authenticator#example-1

Reply:
Done.